pkg/controllers/clusterresourceplacementwatcher/watcher_integration_test.go

```diff
@@ -58,33 +58,19 @@ var _ = Describe("Test ClusterResourcePlacement Watcher", func() {
 		By("By creating a new clusterResourcePlacement")
 		crp := clusterResourcePlacementForTest()
```

Comment:
why not assign this to createdCRP so we don't need to create it again when we delete it?